tasks/devcontainer.py

```diff
@@ -29,6 +29,7 @@
 class SkaffoldProfile(Enum):
     KIND = "kind"
     MINIKUBE = "minikube"
+    NONE = None
```

Comment:
```suggestion
```

Reply:
if I do that I got the error describe in the PR description

```
  File "/workspaces/datadog-agent/tasks/devcontainer.py", line 134, in setup
    configure_skaffold(devcontainer, SkaffoldProfile(SkaffoldProfile))
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not callable
```

to make the linter happy, the function is now using this enum value in a switch case.